products/bigquerydatatransfer/api.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: BigqueryDataTransfer
+display_name: BigQueryDataTransfer
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://bigquerydatatransfer.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/bigquery
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: BigQueryDataTransfer API
+    url: https://console.cloud.google.com/apis/api/bigquerydatatransfer.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Config'
+    base_url: projects/{{project}}/locations/{{location}}/transferConfigs
+    self_link: "{{name}}"
+    update_verb: :PATCH
+    retry_predicates: ["iamMemberMissing"]
```

Comment:
Should this live in `terraform.yaml` instead? It'll only apply to TF and function names aren't formatted the same between languages.